test/io-tests/configs/db-settings-prefix.config

```diff
@@ -1,22 +1,23 @@
-db-anon-role = "postgrest_test_author"
+db-uri = "required"
+db-anon-role = "postgrest_test_anonymous"
 db-channel = "pgrst"
 db-channel-enabled = false
-db-extra-search-path = "public,extensions"
+db-extra-search-path = "public"
 db-max-rows = 1000
```

Comment:
I think it would be valuable to set all values in this config to a non-default value to make sure that the final value that is dumped does really result from the config and not the default fallback.

Given that, I think you should be able to re-use the `no-defaults.config` file here - this does exactly that. If we use `true` as the default for `db-settings` (or whatever name we choose), the `no-defaults` would then have false. You can then override this in python via `PGRST_DB_SETTINGS` env variable.

So basically:
- with `no-defaults.config` and `db-settings=false` the expected output is `expected/no-defaults.config`
- with `no-defaults.config` and `db-settings=true` the expected output is `expected/db-settings.config`

Makes sense?

Reply:
Got it. I'm now using the `no-defaults.config` for the test.